my-app/src/app/screens/Game/index.js

```diff
@@ -55,7 +32,7 @@ class Game extends Component {
     if (winner) {
       status = 'Winner: ' + winner;
     } else {
-      status = 'Next player: ' + (this.state.xIsNext ? 'X' : 'O');
+      status = 'Next player: ' + (this.props.xIsNext ? 'X' : 'O');
```

Comment:
I wonder if this `winner` logic can also be moved to redux. What do you think ? This way, we will make this render a bit cleaner by only returning JSX